app/Helpers/TerrafundDashboardQueryHelper.php

```diff
@@ -2,11 +2,17 @@
 
 namespace App\Helpers;
 
+use App\Models\V2\Projects\Project;
+
 class TerrafundDashboardQueryHelper
 {
-    public static function buildQueryFromRequest($query, $request)
+    public static function buildQueryFromRequest($request)
     {
+        $query = Project::query();
         $query = $query->where('framework_key', 'terrafund');
+        $query = $query->whereHas('organisation', function ($query) {
```

Comment:
I left this feedback on the other PR as well, and it's a minor point, but this would be a little more idiomatic not saving off the query value on each step
```
$query = Project::where('framework_key', 'terrafund')
  ->whereHas('organisation', function ($query) {
```